app/modules/individuals/schemas.py

```diff
@@ -81,7 +84,11 @@ class ElasticsearchIndividualSchema(ModelSchema):
         attribute='names',
         many=True,
     )
+    encounters = base_fields.Nested(ElasticsearchEncounterSchema, many=True)
```

Comment:
To me it made most sense to use the Encounter ES schema to keep a consistent output :man_shrugging: